docs/concepts/automatic-data-locality.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Automatic data locality"
+linkTitle: "Automatic data locality"
+weight: 1
+---
+
+## Ondat volumes move with their pods
+
+Having an Ondat volume instance on the same node as the pod using it can bring some benifits to your application. Unfortunatly pods can move around over time leaving the Volume behind.
```

Comment:
~benifits~ benefits

Reply:
~Unfortunatly~ unfortunately

Reply:
"can bring some benifits to your application": might be good to list them explicitly. I guess the main driver is performance. If the pod is local to the data, then there's no need for network hops

Reply:
Fixed the typos (surprised we don't have a linter that picks these up)
And rewrote the sentence into a mix so it explicitly says "... can improve the performance of your application."